### PR TITLE
Add video teaching the first two parts of the education toolkit

### PR DESCRIPTION
**Pull request by VioletteLepercq**
https://www.youtube.com/watch?v=k8sHYMeDitQ

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,6 +55,14 @@ Would you like to find the tutorials in other languages? You can find all the tr
 >
 > 👉 [click here to access the tutorial](https://colab.research.google.com/github/huggingface/education-toolkit/blob/main/03_getting-started-with-transformers.ipynb)
 
+## **Our Teaching Guide: A Tour Through The 🤗 Hub & Gradio**
+
+In this video, Nate and Lewis summarize the importance of transformers and transfer learning and talk about open science efforts and tools to enable people to work collaboratively in their Machine Learning projects.
```
Comment (lewtun) [inline README.md:60]:
```suggestion
In this video, Nate and Lewis give you a guided tour of Transformers and transfer learning, along with an overview of Hugging Face's open science efforts and tools that enable people to work collaboratively in their Machine Learning projects.
```

**Review by lewtun — APPROVED** [review]
Thanks for adding the video and link @VioletteLepercq ! I left a small nit, but this LGTM 🚀 !

File: README.md (hunk 1 above)
Comment (VioletteLepercq) [reply README.md]: I do not know if I succeeded in embedding the video correctly. Can you confirm that it works?

**Comment by lewtun** [comment]
Video link looks good :)

Screen Shot 2022-04-26 at 17 27 29

**Comment by VioletteLepercq** [comment]
Cool if the video is well embedded within the education toolkit then :-) thanks, @lewtun. Let me know. when it will be ready to be merged :)

**Comment by osanseviero** [comment]
I don't think this shows in markdown though, see https://github.com/huggingface/education-toolkit/blob/08c3bf65bc4da2022880c639fae0994770412364/README.md

**Comment by osanseviero** [comment]
You can try https://github.com/marcomontalbano/video-to-markdown#:~:text=While%20its%20not%20possible%20to,video%20url%20on%20the%20image for example

**Comment by osanseviero** [comment]
As per https://github.com/marcomontalbano/video-to-markdown#:~:text=While%20its%20not%20possible%20to,video%20url%20on%20the%20image for example it's not possible, but you can embed as suggested there at least, that should redirect to the video

**Comment by VioletteLepercq** [comment]
@osanseviero I have tried both of the following ways but the image is not displayed properly. I do not understand why.

[A Tour Through The Hugging Face Hub & A Hands on Guide To Gradio]({https://www.youtube.com/embed/k8sHYMeDitQ} "A Tour Through The Hugging Face Hub & A Hands on Guide To Gradio")


<a href="{https://www.youtube.com/embed/k8sHYMeDitQ}" title="A Tour Through The Hugging Face Hub & A Hands on Guide To Gradio"> A Tour Through The Hugging Face Hub & A Hands on Guide To Gradio </a>

**Comment by osanseviero** [comment]
I just commited a fix. Let me know if it looks good and we can merge :rocket:

**Review by osanseviero — APPROVED** [review]

**Pull request merged**